### ISSUE TITLE
Need to make styling consistent through all the views

### ISSUE THREAD

**Issue by horlorlahdeh** [steem-engine-exchange/steem-engine-dex#201]
Currently, some views look quite different from others in terms of colors and layout;
Need to make styling consistent throughout

**Comment by Vheissu** [comment]
@horlorlahdeh Agreed. Furthermore, I think we need to move away from the bright green, red and blue colours. I think specifically, the buttons in things like modals need some design loving as well. What do you think?

**Comment by horlorlahdeh** [comment]
Sure, @Vheissu some buttons should also be converted to icon-based buttons instead. Will add that to my todo list.
